assets/style.css

```
@@ -19,9 +19,22 @@ body {
 	justify-content: center;
 	align-items: center;
 	position: absolute;
-	top: 0;
+	top: 5vw;
 	bottom: 0;
 	left: 5;
 	right: 5;
 	padding: 5vw;
 }
+
+#info {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	position: absolute;
+	top: 0;
+	bottom: 75;
+	left: 0;
+	right: 0;
+	padding: 3vw;
+	font-size: 3vw;
+}
```

I'm really not sure, but I'm wondering if this would look better with a light font. Maybe something like 
```suggestion
    font-weight: 400;
}
```

---

Lighter font looks good, but 400 looks too thin. Using 500 for the info field.